plugins/core/gs-appstream.c

```diff
@@ -601,9 +601,17 @@ gs_appstream_refine_app_content_rating (GsPlugin *plugin,
 	g_autoptr(AsContentRating) cr = as_content_rating_new ();
 	g_autoptr(GError) error_local = NULL;
 	g_autoptr(GPtrArray) content_attributes = NULL;
+	const gchar *content_rating_kind = NULL;
 
 	/* get kind */
-	as_content_rating_set_kind (cr, xb_node_get_attr (content_rating, "type"));
+	content_rating_kind = xb_node_get_attr (content_rating, "type");
+	/* we only really expect/support OARS 1.0 and 1.1 */
+	if (content_rating_kind == NULL ||
+	    !g_str_has_prefix (content_rating_kind, "oars-1.")) {
```

Comment:
This check is wrong – we don't understand a future `oars-1.2` format.

Reply:
I know, this was also existing code - I just moved the conditionals around to avoid a warning when checking against a `NULL` type. I will fix the conditional though to only check 1.0 and 1.1.

Reply:
Oh! I missed that it's existing code.

Reply:
Upstreamed as https://gitlab.gnome.org/GNOME/gnome-software/merge_requests/344.